### ISSUE TITLE
Binary package files missing plotting script

### ISSUE THREAD

**Issue by graeme-a-stewart** [HSF/prmon#70]
I wanted to make a static binary tarball that could be used to easily distribute prmon without needing to build it. Unfortunately it seems the plotting script is missing from the files generated by `make package`:

```
[root@58bab5ac49c3 prmon]# tar -tvzf prmon_0.1.0_x86_64-centos7-gnu72-opt.tar.gz 
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/prmon/
-rw-r--r-- root/root      1409 2018-06-11 07:26 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/prmon/prmonConfig.cmake
-rw-r--r-- root/root      3198 2018-06-11 07:26 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/prmon/prmonTargets.cmake
-rw-r--r-- root/root      1607 2018-06-11 07:26 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/prmon/prmonConfigVersion.cmake
-rw-r--r-- root/root       759 2018-06-11 07:26 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/cmake/prmon/prmonTargets-release.cmake
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/doc/
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/doc/prmon/
-rw-r--r-- root/root     11357 2018-03-05 13:13 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/doc/prmon/LICENSE
-rw-r--r-- root/root       544 2018-03-05 13:13 prmon_0.1.0_x86_64-centos7-gnu72-opt/share/doc/prmon/NOTICE
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/bin/
-rwxr-xr-x root/root   2348928 2018-06-11 07:47 prmon_0.1.0_x86_64-centos7-gnu72-opt/bin/prmon
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/include/
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/include/prmon/
-rw-r--r-- root/root        81 2018-06-11 07:26 prmon_0.1.0_x86_64-centos7-gnu72-opt/include/prmon/prmonVersion.h
drwxr-xr-x root/root         0 2018-06-11 07:48 prmon_0.1.0_x86_64-centos7-gnu72-opt/lib64/
```

**Comment by amete** [comment]
I just pushed `master-scripts-cmake-update` which should fix this, can you check @graeme-a-stewart ?

**Comment by graeme-a-stewart** [comment]
Hi

Yes, looks good to me. I see you just made the PR now...

Thanks

**Comment by graeme-a-stewart** [comment]
Fixed in #74
